src/main/kotlin/fi/metatavu/oss/api/impl/SystemApiImpl.kt

```diff
@@ -0,0 +1,21 @@
+package fi.metatavu.oss.api.impl
+
+import fi.metatavu.oss.api.spec.SystemApi
+import javax.enterprise.context.RequestScoped
+import javax.ws.rs.Produces
+import javax.ws.rs.core.Response
+
+/**
+ * System API implementation
+ *
+ * @author Antti Leppä
+ */
+@RequestScoped
+class SystemApiImpl: SystemApi, AbstractApi()  {
+
+    @Produces("application/json")
```

Comment:
Doesn't this produce text/plain instead?

Reply:
Yes, seems to be result of copy paste from the template project. Will fix